.gitignore

```diff
@@ -3,3 +3,5 @@
 # SPDX-License-Identifier: MPL-2.0
 
 .stack-work/
+.idea/
+.DS_Store
```

Comment:
Please, don’t do this. Do this: https://www.notion.so/serokell/gitignore-3d45b853c4494fa282dc04269f851b2e.